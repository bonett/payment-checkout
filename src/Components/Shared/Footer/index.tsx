/* eslint-disable */
import React from "react";

function Footer() {
  return (
    <footer className="footer-content">
      <div className="container">
        <div className="footer-content_wrapper">
          <p className="footer-content_text">Created by Wilfrido Bonett - Copyright &copy; {new Date().getFullYear()}</p>
        </div>
      </div>
    </footer>
  );
}

export default Footer;
