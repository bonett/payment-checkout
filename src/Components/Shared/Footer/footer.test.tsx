import React from 'react';
import { render, screen } from '@testing-library/react';
import Footer from './';

test('renders Footer', () => {
  render(<Footer />);
  const linkElement = screen.getByText(/Created by Wilfrido Bonett - Copyright Â© 2024/i);
  expect(linkElement).toBeInTheDocument();
});
