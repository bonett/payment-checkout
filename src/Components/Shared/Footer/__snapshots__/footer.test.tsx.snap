// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Should renders footer componente 1`] = `
<div>
  <footer
    class="footer-content"
  >
    <div
      class="container"
    >
      <div
        class="footer-content_wrapper"
      >
        <p
          class="footer-content_text"
        >
          Created by Wilfrido Bonett - Copyright Â© 
          2024
        </p>
      </div>
    </div>
  </footer>
</div>
`;
